router/keeper/keeper.go

```diff
@@ -27,11 +28,19 @@ type Keeper struct {
 	distrKeeper    types.DistributionKeeper
 }
 
+func TransferDefaultTimeout(ctx sdk.Context) uint64 {
```

Comment:
I know we were setting the timeout to 30 minutes before but what if we used the same defaults from ibc-go just for consistency?

https://github.com/cosmos/ibc-go/blob/06c2d764df5945d8de93870184ba38b7a8a5dc05/modules/apps/transfer/types/packet.go#L11-L21

Reply:
I also used the height timeout from IBC then. Please check!

Reply:
Looks good to me! Thanks for that 🙂